### ISSUE TITLE
Integrate REV Arm Code

### ISSUE THREAD

**Issue by Spenceroni-Pepperoni** [Robotiators-888/2023Season#12]
**Is your feature request related to a problem? Please describe.**
The setpoints from the rev starter code are really useful and the work is already done for us. This should be after #10 is closed

**Describe the solution you'd like**
I want the features integrated in our code and instead of a,b,x,y buttons translate it to the equal buttons on the D-Pad of the controller

**Additional context**
The type of button should be a POVButton
